src/main/windows/widget.ts

```diff
@@ -121,8 +99,19 @@ function getLocationUnderTray(
   };
 }
 
-eventBus.on('APP_IS_READY', async () => {
-  await createWidget();
+export function setBoundsOfWidgetByPath(
+  widgetWindow: BrowserWindow,
+  tray: TrayMenu
+) {
+  const { placeUnderTray, ...size } = widgetConfig;
 
-  eventBus.emit('WIDGET_IS_READY');
-});
+  const bounds = tray.bounds;
+
+  if (placeUnderTray && bounds) {
+    const location = getLocationUnderTray(size, bounds);
+    widgetWindow.setBounds({ ...size, ...location });
+  } else {
```

Comment:
This is a case we could simplify (from the comment above). I belive the window was only centered when it was displaying the login page

Reply:
agreed, will remove this